### ISSUE TITLE
lANDI mistakenly says elements do not have href when they do

### ISSUE THREAD

**Issue by ollie-iterators** [SSAgov/ANDI#135]
<img width="1155" alt="2022-04-24_15-25-45" src="https://user-images.githubusercontent.com/46904826/164993124-c26ce65f-8eff-45c1-ac8b-a707eae62ded.png">

As seen in this image, on boston.gov, the code says that the element does not have href, id, or tabindex when href is visible on the ANDI screen.

**Comment by fstrr** [comment]
It is likely confused because you've used a `role="group"` attribute on the `<a>` element to change the semantics of the link. By adding `role="group"`, you're saying that this isn't a link, it's a group. If you remove the role="group" attribute from the link then the problem goes away. If you test the page with a screenreader (I've just tried VoiceOver on macOS), you'll find that it doesn't report any of those links as links because of the `group` role, which you don't need.

**Comment by JohnCotterSSA** [comment]
Good catch, @fstrr , I was just about to comment on the role=group. Indeed, a screen reader would not recognize this element as a link.

The ANDI alert in this instance is misleading. We'll put this in the backlog.
